recipes-core/images/nilrt-image-common.inc

```diff
@@ -11,4 +11,19 @@ IMAGE_PREPROCESS_COMMAND += "rootfs_update_timestamp;"
 # Do not install license subpackages if they are only recommended.
 BAD_RECOMMENDATIONS_pn-${PN} += "*-lic"
 
+ROOTFS_POSTPROCESS_COMMAND += "\
+       ${@oe.utils.conditional('DISTRO', 'nilrt-nxg', '', 'move_kernel;', d)} \
+"
```

Comment:
The `nilrt-nxg` DISTROs are all deprecated at this point. We can avoid this conditional, I think.